{"uuid":"45b16cef-66c5-4b8c-afd3-12a40a5e9f12","name":"Scenario Outline: Verify that user receives error for invalid python code for Max Consecutive Ones Question","children":["f53f256b-46d2-406d-81b8-151cdea20cef"],"befores":[{"name":"hooks.HooksPage.setUp()","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1747066319284,"stop":1747066321555}],"afters":[{"name":"hooks.HooksPage.tearDown(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1747066324632,"stop":1747066324806}],"start":1747066319284,"stop":1747066324806}